tools/run_mview_sperson_estimator.py

```diff
@@ -0,0 +1,217 @@
+                        os.path.basename(img_list_path).rsplit(
+                            '.', 1)[0].split('_')[-1])
+                    logger.info(f'Visualizing {smpl_type.upper()}' +
+                                f' from view {view_idx:02d}.')
+                    with open(img_list_path, 'r') as f_read:
+                        lines = f_read.readlines()
+                    img_path = lines[0].strip()
+                    img_dir = os.path.join(args.data_root,
+                                           os.path.dirname(img_path))
+                    fisheye_param = cam_param_list[idx]
+                    pinhole_param = PinholeCameraParameter(
+                        K=fisheye_param.intrinsic,
+                        R=fisheye_param.extrinsic_r,
+                        T=fisheye_param.extrinsic_t,
+                        height=fisheye_param.height,
+                        width=fisheye_param.width,
+                        convention=fisheye_param.convention,
+                        world2cam=fisheye_param.world2cam,
+                        logger=logger)
+                    visualize_smpl_data(
+                        smpl_data=smpl_data,
+                        body_model=body_model,
+                        cam_param=pinhole_param,
+                        output_path=os.path.join(
+                            scene_vis_dir,
+                            f'{smpl_type}_data_projected_{view_idx:02d}.mp4'),
+                        overwrite=True,
+                        background_dir=img_dir,
+                        disable_tqdm=False,
+                        logger=logger,
+                        device=body_model.betas.device)
+                    break  # TODO: remove this break
+
+
+def setup_parser():
+    parser = argparse.ArgumentParser(
+        description='Run MultiViewSinglePersonSMPLEstimator on a dataset.')
+    # input args
+    parser.add_argument(
+        '--data_root',
+        type=str,
+        help='Path to the root directory of' + ' the mview sperson dataset.',
+        default='')
+    parser.add_argument(
+        '--meta_path',
+        type=str,
+        help='Path to the meta-data dir.' +
+        ' Camera parameters and image paths will be read from here.' +
+        ' Estimation results will be written here.',
+        default='')
+    # model args
+    parser.add_argument(
+        '--estimator_config',
+        help='Config file for MultiViewSinglePersonSMPLEstimator.',
+        type=str,
+        default='configs/humman_mocap/mview_sperson_smpl_estimator.py')
+    # output args
+    parser.add_argument(
+        '--visualize',
+        action='store_true',
+        help='If checked, visualize result.',
+        default=False)
+    # log args
+    parser.add_argument(
+        '--disable_log_file',
+        action='store_true',
+        help='If checked, log will not be written as file.',
+        default=False)
+    args = parser.parse_args()
+    return args
+
```

Comment:
Shall we add `--output_path`, and save keypoints2d_list, keypoints3d, smpl_data in `output_path` rather than `scene_path`.

Reply:
Added in [83ac66b](https://github.com/openxrlab/xrmocap/pull/123/commits/83ac66b600c1681e720382df169657ac87ed3b88).